docs/fusion.md

```diff
@@ -101,6 +101,11 @@ Seqera Platform is able to automatically format and configure the NVMe instance
 the option "Use Fast storage" when creating the Batch compute environment.
 :::
 
+:::{tip}
```

Comment:
I think it's important to stress that it's a good alternative but not as good as NVMe disk.

"While not as fast as NVMe storage, an EBS gp3 volume with a throughput of 325 MiB/s or more and a size of 100 GiB or more provides sufficient performance for many workloads on your compute node"